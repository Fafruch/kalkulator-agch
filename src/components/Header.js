/**
 * Created by Fafruch on 18/07/2017.
 */

import React from 'react'

const Header = () => (
  <div style={{ textAlign: 'center' }}>
    <img
      src='agh_logo.jpg'
      alt='Logo AGH'
      style={{ height: 200 }}
    />
    <h1>Kalkulator </h1>
  </div>
)

export default Header
