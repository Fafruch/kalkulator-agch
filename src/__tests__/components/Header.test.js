import React from 'react'
import { shallow } from 'enzyme'

import Header from '../../components/Header'

const setup = (setupProps = {}) => {
  const defaultProps = {}
  const props = { ...defaultProps, ...setupProps }
  const wrapper = shallow(<Header {...props} />)

  return {
    props,
    wrapper
  }
}

describe('Header component', () => {
  test('should render', () => {
    const { wrapper } = setup()

    expect(wrapper).toMatchSnapshot()
  })
})
