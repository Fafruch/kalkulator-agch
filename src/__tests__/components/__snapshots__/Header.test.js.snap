// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="Logo AGH"
          src="agh_logo.jpg"
/>,
        <h1>
          Kalkulator
</h1>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "alt": "Logo AGH",
          "src": "agh_logo.jpg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Kalkulator",
        },
        "ref": null,
        "rendered": "Kalkulator",
        "type": "h1",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="Logo AGH"
            src="agh_logo.jpg"
/>,
          <h1>
            Kalkulator
</h1>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "alt": "Logo AGH",
            "src": "agh_logo.jpg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Kalkulator",
          },
          "ref": null,
          "rendered": "Kalkulator",
          "type": "h1",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
